_pages/policies.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: Policies
+subtitle: Policies
+permalink: /policies/
+---
+
+<div class="pretty-links">
+
+## Territorial Acknowledgement ##
+
+StarCon would like to acknowledge that we are on the traditional territory of
+the Neutral, Anishnawbe and Haudenosaunee peoples. The University of Waterloo is
+situated on the Haldimand Tract, the land promised to the Six Nations that
+includes six miles on each side of the Grand River. It is important to remember
+that the land that we are using today for this event was stolen, and that there
+is lots of work to do to bring back the rights that indigenous and Native people
+deserve.
```

Comment:
I think we should add something about how  this relates to our mission to be inclusive of people from diverse backgrounds, and that in doing so it's important to recognize this kind of thing. Not totally sure how to word that